app/controllers/api/v1/mc_questions_controller.rb

```diff
@@ -0,0 +1,67 @@
+class Api::V1::McQuestionsController < ApplicationController
+    before_action :set_mc_question, only: [:show, :edit, :update, :destroy]
+    skip_before_action :verify_authenticity_token
+
+    # GET /mc_questions or /mc_questions.json
+    def index
+      @mc_questions = McQuestion.all
+      render json: @mc_questions.to_json(include: :mc_options)
+    end
+  
+    # GET /mc_questions/1 or /mc_questions/1.json
+    def show
+        render json: @mc_question.to_json(include: :mc_options)
+    end
+  
+    # POST /mc_questions or /mc_questions.json
+    def create
+        @mc_question = McQuestion.new(params.require(:mc_question).permit(:question, :survey_id))
```

Comment:
i think you could make a helper method to reduce the code duplication of `params.require(:mc_question).permit(:question, :survey_id)`, the user controller and text question controller have a private method for this

Reply:
will do